securedrop-protocol/src/sign.rs

```diff
@@ -44,3 +44,52 @@ impl VerifyingKey {
             .map_err(|_| anyhow::anyhow!("Signature verification failed"))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use alloc::vec::Vec;
+    use proptest::prelude::*;
+    use rand::thread_rng;
+
+    proptest! {
```

Comment:
Just a drive-by comment to say how happy I am to see property-based testing here!